configs/agent/sample.toml

```diff
@@ -56,11 +56,12 @@ scaling-group = "default"
 # The port number for aiomonitor.
 # aiomonitor-port = 50200
 
-# Explicitly set allowed compute plugins to avoid conflicts of mock &
+# Explicitly set allowed(or blocked) compute plugins to avoid conflicts of mock &
 # open-source CUDA plugins in the mono-repo source installation.
 # Regardless of this config, CPUPlugin and MemoryPlugin are always enabled.
 # example: ["ai.backend.accelerator.cuda_mock"]
 # allow-compute-plugins = []
+# block-compute-plugins = []
```

Comment:
Please fix agent config validation to raise validation error when duplicate value is in both arrays.